src/main/webapp/app/overview/course-exams/course-exams.component.scss

```diff
@@ -6,6 +6,26 @@
     box-shadow: 0 2px 4px 0 var(--success);
 }
 
+.hover-effect:hover {
```

Comment:
this is duplicated code. Can we move this into global.scss to avoid code duplication